cache/cache.go

```diff
@@ -75,15 +74,12 @@ func NewFSCache(cacheDir string) (FSCache, error) {
 	}, nil
 }
 
-func (fs FSCache) GetLayer(layerID string) (types.LayerInfo, error) {
+func (fs FSCache) GetLayer(diffID string) (types.LayerInfo, error) {
 	var layerInfo types.LayerInfo
 	err := fs.db.View(func(tx *bolt.Tx) error {
-		// get a decompressed layer ID
-		decompressedBucket := tx.Bucket([]byte(decompressedDigestBucket))
-		layerBucket := tx.Bucket([]byte(layerBucket))
-
 		var err error
-		layerInfo, err = fs.getLayer(decompressedBucket, layerBucket, layerID)
+		layerBucket := tx.Bucket([]byte(layerBucket))
```

Comment:
We should check what `layerBucket` has before we use it. If it's `nil`, `fs.getLayer` will panic.

Reply:
Is your question duplicated? Or, do you mean we should check it in any case?
https://github.com/aquasecurity/fanal/pull/96#discussion_r404431270

Reply:
I think it's OK. It is a potential for panic but since we already make sure NewFsCache creates it, as long as we call NewFsCache before using FsCache{} it should be fine.

Reply:
OK. If we face a panic, it is the time when we should handle it.